src/main/java/org/jenkinsci/plugins/workflow/steps/durable_task/WindowsPowerShellScriptStep.java

```diff
@@ -32,11 +32,11 @@
 /**
  * Asynchronous batch script execution.
  */
-public class PowershellScriptStep extends DurableTaskStep {
+public class WindowsPowerShellScriptStep extends DurableTaskStep {
```

Comment:
Please don't change class names or move files (unless absolutely necessary) to avoid issues with backwards compatibility (for example, I found [this reference](https://github.com/jenkinsci/labelled-steps-plugin/blob/a7015e81cb60f0adb5baad170412c730bfe2577a/src/main/java/me/vickychijwani/jenkins/LabelledPowerShellStep.java#L70) to the existing class in a plugin). I would recommend reverting the moves/renames and leaving only the change in `getDisplayName`.

Reply:
Fair enough. Only thought to do these branding fixes because I saw another rename come in when I updated my fork of this repo. But looking now, that was only a case fix in a resource folder name, not a class rename.